### DISCUSSION TITLE
Style not works in dynamic class

### DISCUSSION THREAD

**Discussion by saddit** [tailwindlabs/tailwindcss#7144] [Help] [answered]
```vue
<button
    :class="getColor(act)"
  >{{ act.name }}</button>
```
```js
function getColor(act: EntityAction) {
  return `text-${act.color ?? 'blue'}-400`
}
```
![image](https://user-images.githubusercontent.com/55338151/150349557-aa69d218-234a-4621-be39-04ed22e1fe16.png)

Class name is showing in attribute correctly but it doesn't import style

Changing function to return `act.color ? 'text-red-400' : 'text-blue-400'` can work perfectly

What's the problem and why is that?

**Comment by NovusOpus** [comment, marked as answer]
That's explained here in the [Docs](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
